src/view/popup/utils/backgroundStopRecordingBytes.ts

```diff
@@ -1,13 +1,12 @@
-export const backgroundStopRecordingBytes = () => {
-    chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
-        if (tabs.length > 0) {
-            const tabId = tabs[0].id;
-            if (tabId) {
-                chrome.runtime.sendMessage({
-                    command: "stopRecordingBytesTransferred",
-                    tabId,
-                });
-            }
+export const backgroundStopRecordingBytes = async () => {
+    const tabs = await chrome.tabs.query({ active: true, currentWindow: true });
+    if (tabs.length > 0) {
+        const tabId = tabs[0].id;
```

Comment:
why just the first one ?

Reply:
the first one is the active tab - where the chrome extension is open, pushed a new commit for readability